docs/projects/version-control/index.md

```diff
@@ -5,68 +5,71 @@ position: 110
 hideInThisSection: true
 ---
 
-## Introduction 
+![TODO: Add hero image](todo.png) 
 
-The Configuration as Code (config-as-code) feature adds support for configuring Octopus projects to store project resources in a Git repository. For now, only the _deployment process_, _deployment settings_, and _non-sensitive variables_ can be version-controlled.
+Config as Code in Octopus delivers the full power of Git with the usability of Octopus. We support a rich two-way sync which means your Git repo is the source of truth, and full support to version control and to evolve your configuration variables alongside your deployment process and application code. You get all the benefits of config as code for your application and infrastructure configuration that change as you deploy through your environments. 
 
-The Octopus UI needed to remain fully functional for version-controlled projects, and it has. You can continue to use the UI exactly as you always have, but with an additional superpower: Git branches are now exposed in the UI, allowing editing of currently supported project configuration on any branch via the UI. If you type the name of a branch that doesn't exist in your repository, you'll see an option to create that branch. This option is available when committing changes too.
+When Config as Code is enabled for a project, you can continue to use the Octopus UI, or edit the text files in your favorite editor and command line tools. This gives you the freedom to edit, find and replace and perform bulk changes with regex replacements where you choose. Manage variables where you are most productive, branching, committing to and deploying from branches, as well as pull request workflows to review and approve changes. 
 
-![Branch-switcher UI](branch-switcher-ui.png "width=500")
+## Why use Config as Code?
```

Comment:
Should we have a one sentence introducing why to use config as code, rather than go straight from a H2 to H3 header?